src/helpers/getRandomImage.js

```diff
@@ -0,0 +1,26 @@
+import challenge1 from "../images/challenge1.png";
+import challenge2 from "../images/challenge2.png";
+import challenge3 from "../images/challenge3.png";
+import challenge4 from "../images/challenge4.png";
+import challenge5 from "../images/challenge5.png";
+import challenge6 from "../images/challenge6.png";
+import challenge7 from "../images/challenge7.png";
+import challenge8 from "../images/challenge8.png";
+
+const challengeImages = [
+  challenge1,
+  challenge2,
+  challenge3,
+  challenge4,
+  challenge5,
+  challenge6,
+  challenge7,
+  challenge8,
+];
+
+export const getRandomImage = () => {
+  const imagesAmount = challengeImages.length;
+  const randomIndex = Math.floor(Math.random() * imagesAmount);
```

Comment:
the random image should be changed after refreshing the page.
it is an example from prev. project, but you can just modify it

```
export const GROUP_ID = -1;

export default function getRandomColor(id: number) {
  if (id === GROUP_ID) return '#317ad8';

  const lastDigit = id % 10;
  const colorsRange = ['#677d96', '#e169b1', '#eb8c57', '#f2b94a', '#3591c5', '#697ce1'];
  const arrIndex = lastDigit > colorsRange.length ? lastDigit - (colorsRange.length + 1) : lastDigit;
  return colorsRange[arrIndex];
}

```